src/common/settings.js

```diff
@@ -623,6 +623,12 @@ const character_settings = {
         "description": "The luck of your people guides your steps",
         "type": "bool",
         "default": true
+    },
+    "empowered-evocation": {
+        "title": "Evocation Wizard - Empowered Evocation",
+        "description": "Your prowess in Evocation lends power to your Evocation spells",
+        "type": "bool",
+        "default": true
     }
```

Comment:
adding option at the end of the list? not caring about sort order?  😱 What a monster!